### ISSUE TITLE
Spurious "Probably illegal use of (Int,Int) inside curly."

### ISSUE THREAD

**Issue by sbromberger** [tonyhffong/Lint.jl#83]
```
readedges = Set{(Int,Int)}()
```

I know it hedges with "probably", but is there something wrong with this code?

**Comment by tonyhffong** [comment]
Nothing. It's definitely a false positive.
